### PR TITLE
feat : comments api, and single post api

### PR DESCRIPTION
**Pull request by mikesch803**
1. add single post api
2. add comments api (add, edit, delete, getComments)

##  [https://deploy-preview-8--react-connect-sm.netlify.app/home](https://deploy-preview-8--react-connect-sm.netlify.app/home)

### CODE REVIEW

**Review by VLeads — APPROVED** [review]
Looks good. Nothing extra to suggest.

File: src/components/edit-profile/EditProfileModal.jsx [hunk 1]
```diff
@@ -24,8 +24,16 @@ export function EditProfileModal({profileUser}) {
   const handleOpen = () => setOpen(true);
   const handleClose = () => setOpen(false);
   const dispatch = useDispatch()
-const [editProfile, setEditProfile] = React.useState({...profileUser})
```
Comment (Hamza-777) [inline src/components/edit-profile/EditProfileModal.jsx:27]: Use the following kind of import it is more clean:
import React, { useState, useEffect } from 'react';

then you can avoid using React.useState and React.useEffect everywhere.

**Review by Hamza-777 — APPROVED** [review]
Nice work.

File: src/components/edit-profile/EditProfileModal.jsx (hunk 1 above)
Comment (mikesch803) [reply src/components/edit-profile/EditProfileModal.jsx:27]: sure, i will fix this in next PR

**Pull request merged**